### ISSUE TITLE
Some ideas for mod

### ISSUE THREAD

**Issue by Radviger** [lumien231/Custom-Main-Menu#35]
Hi, lumien231!
I have a few good ideas for your mod that I want to see in my menu :3
There is:
1) Player face as placeholder for image element
2) Placeholder that count players on exact server
3) Component panels that can be moved or aligned with contained elements
4) Some slideshow effects like in MS PowerPoint

**Comment by lumien231** [comment]
Sorry for the late answer, most of these things aren't really possible with the current design. Since i wasn't intending to add this much stuff to this mod at the beginning it's not written very "dynamic". I am going to rewrite the mod in the future and hopefully i can add then all these things :).

**Comment by shaharcoolmen** [comment]
i really like the function method

**Comment by killjoy1221** [comment]
It might be better to just make actions an array.
```json
"actions": [
  {
    "type": "openGui",
    "gui": "muliplayer"
  },
  {
    "type": "playSound",
    "sound": "mob.zombie.say"
  }
]
```

**Comment by lumien231** [comment]
Yeah that wouldn't be too hard, but realistically which actions would you want to execute like that? For Sounds there already is pressSound

**Comment by kreezxil** [comment]
I know this is really necro, but it is open and I would like to see relative positioning of the elements, that way I don't have to various file formats for the different dimensions.

Also have you implemented any of the stuff you marked as "future"?

And ... Chrome doesn't like the mixed mod stuff that is going on in the Pages section of the CurseForge pages. You need to reinstall all the videos and make sure they working properly.

**Comment by lumien231** [comment]
You generally don't need different files for different dimensions, that's what alignments are meant for. Did you try using those? Will fix the pages.

**Comment by kreezxil** [comment]
I found the undocumented left alignment which is helping greatly.
